### DISCUSSION TITLE
7C1645AC-07F5-48C8-9FD4-E4A5CC34A5E9

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#20] [General]
# Kepler-70

Kepler-70 is a blue subdwarf about 4,155 light-years away in the constellation Cygnus. Observations in 2011 show that two very hot subterran planets are present in tight orbits less than 1.2 million km from the star's photosphere. However, subsequent analysis of the system in 2019 suggest that the supposed planetary signatures are actually stellar activity, thus disproving their existence.

https://celestia.mobi/addon?item=7C1645AC-07F5-48C8-9FD4-E4A5CC34A5E9

**Comment by exocritic5457** [comment]
both planet's textures have seams around the daysides

**Reply by m51uls1b** [reply]
ok
